fractal_tasks_core/cellpose_segmentation.py

```diff
@@ -505,7 +708,17 @@ def cellpose_segmentation(
             )
             channels = [0, 0]
 
-        fov_mask = segment_FOV(
+        img_np, background_3D, current_label = preprocess_cellpose_input(
+            image_array=img_np,
+            use_masks=use_masks,
+            region=region,
+            # primary_label_path=f"{zarrurl}labels/{primary_label_name}/0",  # FIXME: which level? # noqa
+            current_label_path=f"{zarrurl}labels/{output_label_name}/0",  # FIXME: which level? # noqa
+            ROI_table_path=f"{zarrurl}tables/{ROI_table_name}",
+            ROI_index=i_ROI,
+        )
+
+        new_mask = segment_ROI(
```

Comment:
The output is not a mask, but a label image, right? e.g. new_label_img

Reply:
Agreed, it's more clear.
(confusion came from the fact that labels -e.g. the output of cellpose `eval`- are called masks)